src/microns_to_nwb/tools/behavior/behavior.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+from hdmf.backends.hdf5 import H5DataIO
+from phase3 import nda
+from pynwb import TimeSeries
+from pynwb.behavior import PupilTracking, SpatialSeries, EyeTracking
+
+
+def add_eye_tracking(scan_key, nwb):
+    pupil_min_r, pupil_maj_r, pupil_x, pupil_y, timestamps = (nda.RawManualPupil() & scan_key).fetch1(
+        "pupil_min_r", "pupil_maj_r", "pupil_x", "pupil_y", "pupil_times"
+    )
+
+    pupil_min_r = TimeSeries(
+        name="pupil_min_r",
+        description="minor axis of pupil tracking ellipse",
+        data=H5DataIO(pupil_min_r, compression=True),
+        timestamps=H5DataIO(timestamps, compression=True),
+        unit="n.a.",
+    )
+
+    pupil_maj_r = TimeSeries(
+        name="pupil_maj_r",
+        description="jajor axis of pupil tracking ellipse",
```

Comment:
```suggestion
        description="Major axis of pupil tracking ellipse.",
```

Minor typo - I also always prefer proper sentences in descriptions